src/workflows/airqo_etl_utils/ml_utils.py

```
@@ -741,3 +765,111 @@ def save_faulty_devices(*dataframes):
+        #         stds.append(testset[t].std())
+        #         score = mean_squared_error(pred, testset[t], squared=False)
+        #         mlflow.log_metric("rmse", score)
+        #         mlflow.sklearn.log_model(model, "model")
+        #
+        # for v_train, v_test in cv.split(data, groups=groups):
+        #     train_v, test_v = data.iloc[v_train], data.iloc[v_test]
+        #     origin = data.iloc[v_test]
+        #     rmse.append(validate(train_v, test_v, 'pm2_5', origin))
+
+        GCSUtils.upload_trained_model_to_gcs(
+            model, project_id, bucket, f"satellite_prediction_model.pkl"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)